app/models/price_group.rb

```diff
@@ -78,4 +86,63 @@ def external?
     !is_internal?
   end
 
+  private
+
+  # Find a global price group by name, or create it if it does not exist
+  def self.find_or_create_global(name:, is_internal: false, admin_editable: true)
+    global_price_groups = PriceGroup.globals
+    found_price_group = global_price_groups.find_by(name:)
+
+    if found_price_group
+      log_string = "Global price group '#{name}' already exists."
+
+      puts log_string
+      logger.info log_string
```

Comment:
Do we need the explicit logging here and in other methods related to setup_global?  I think that method will be called from the console or rake task to `puts` should do the trick.